user-org-actor/src/main/java/org/sunbird/user/service/AddressServiceImpl.java

```diff
@@ -0,0 +1,75 @@
+package org.sunbird.user.service;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.DaoImplType;
+import org.sunbird.exception.BaseException;
+import org.sunbird.request.Request;
+import org.sunbird.response.Response;
+import org.sunbird.user.dao.IUserAddressDao;
+import org.sunbird.user.dao.UserDaoFactory;
+import org.sunbird.util.ProjectLogger;
+import org.sunbird.util.jsonkey.JsonKey;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AddressServiceImpl implements IAddressService {
+    @Override
+    public Response insertAddress(Request request) throws BaseException {
+        return insert(request);
+    }
+
+    private Response insert(Request request) {
+        Map<String, Object> requestMap = request.getRequest();
+        String userId = (String)requestMap.get(JsonKey.USER_ID);
+        List<Map<String, Object>> addressList =
+                (List<Map<String, Object>>) requestMap.get(JsonKey.ADDRESS);
+        Response response = new Response();
+        List<String> errMsgs = new ArrayList<>();
+        List<Map<String, Object>> responseAddressList = new ArrayList<>();
+        try {
+            for (int i = 0; i < addressList.size(); i++) {
+                try {
+                    Map<String, Object> address = addressList.get(i);
+                    responseAddressList.add(createAddress(userId, address));
+                } catch (BaseException e) {
+                    errMsgs.add(e.getMessage());
+                    ProjectLogger.log(
+                            "AddressServiceImpl:insert Address: Exception occurred with error message = "
+                                    + e.getMessage(),
+                            e);
```

Comment:
user Logger.Error enum

Reply:
If we will pass exception object it will default take Logger.Error enum, no need to pass this explicitly